com.vscodium.codium-insiders.yaml

```diff
@@ -0,0 +1,145 @@
+      - mv usr/share/codium-insiders /app/share
+        # use sed because because desktop-file-edit can't modify different group keys
+      - sed -e "s/^Icon=.*/Icon=$FLATPAK_ID/" -i usr/share/applications/*.desktop
+      - sed -e "s|/usr/share/codium-insiders/||g" -i usr/share/applications/*.desktop
+        # use install instead of desktop-file-install because can't rename the output file
+      - install -Dm644 usr/share/applications/codium-insiders.desktop /app/share/applications/$FLATPAK_ID.desktop
+      - install -Dm644 usr/share/applications/codium-insiders-url-handler.desktop
+        /app/share/applications/$FLATPAK_ID-url-handler.desktop
+      - install -Dm644 usr/share/mime/packages/codium-insiders-workspace.xml /app/share/mime/packages/$FLATPAK_ID-workspace.xml
+        # Sugested by desktop-file-validate
+      - desktop-file-edit --add-category="Utility" /app/share/applications/$FLATPAK_ID.desktop
+      - rm -r codium.deb control.tar.* data.tar.xz debian-binary usr
+      - mkdir /app/tools
+    sources:
+      - type: file
+        url: https://github.com/VSCodium/vscodium-insiders/releases/download/1.85.0.23310-insider/codium-insiders_1.85.0.23310_amd64.deb
+        sha256: 7b4b90bdb74d7761c4c7ad4f66b189f221d8fd8bff4b1df15cf2b2fe9182aa60
+        dest-filename: codium.deb
+        only-arches:
+          - x86_64
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/VSCodium/vscodium-insiders/releases/latest
+          tag-query: .tag_name
+          version-query: .tag_name
+          timestamp-query: .published_at
+          url-query: '[.assets[] | select(.name|test("codium-insiders_.+_amd64.deb$"))][0]
+            | .browser_download_url'
+      - type: file
+        url: https://github.com/VSCodium/vscodium-insiders/releases/download/1.85.0.23310-insider/codium-insiders_1.85.0.23310_arm64.deb
+        sha256: 44b9331333c94b984ac510d2b7436168c91e4899d226f0edd5d082b3d1da4747
+        dest-filename: codium.deb
+        only-arches:
+          - aarch64
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/VSCodium/vscodium-insiders/releases/latest
+          tag-query: .tag_name
+          version-query: .tag_name
+          timestamp-query: .published_at
+          url-query: '[.assets[] | select(.name|test("codium-insiders_.+_arm64.deb$"))][0]
+            | .browser_download_url'
+      - type: file
+        url: https://raw.githubusercontent.com/VSCodium/vscodium/f1fa60db6d5332bef6cb2fad55b809a48b76179c/icons/insider/codium_cnl.svg
+        sha256: 0cc2ded6ad8109c59bdfad9ef71c0afda0e6d5d6a26aaf03eac5d4a8a92fa4e5
+        dest-filename: codium-insiders.svg
+      - type: file
+        path: com.vscodium.codium-insiders.metainfo.xml
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak.git
+        branch: main
```

Comment:
```suggestion
```

This will cause mismatches if there are new commits to main and the current commit below does not point to it.